napari/_qt/layer_controls/qt_image_controls_base.py

```diff
@@ -80,12 +88,17 @@ def __init__(self, layer: Image):
 
         self.clim_popup = None
 
-        self.contrastLimitsSlider.mousePressEvent = self._clim_mousepress
-        set_clim = partial(setattr, self.layer, 'contrast_limits')
+        # self.contrastLimitsSlider.mousePressEvent = self._clim_mousepress
```

Comment:
can we delete this line and also the self._clim_mousepress method now?

Reply:
Yes. I copy the docstring and clean.